File: 1-Iniciante/1828.c
#include <stdio.h>
#include <string.h>

int main(void)
{
    int qnt = 0, i = 0, num = 0;
    char sheld[10], raj[10];

    scanf("%d", &qnt);

    for (i = 0; i < qnt; ++i)
    {
        scanf(" %s", sheld);
        scanf(" %s", raj);

        if (strcmp(sheld, raj) == 0)
            printf("Caso #%d: De novo!\n", i + 1);
        else if (strcmp(sheld, "tesoura") == 0 && (strcmp(raj, "papel") == 0 || strcmp(raj, "lagarto") == 0))
            printf("Caso #%d: Bazinga!\n", i + 1);
        else if (strcmp(sheld, "papel") == 0 && (strcmp(raj, "pedra") == 0 || strcmp(raj, "Spock") == 0))
            printf("Caso #%d: Bazinga!\n", i + 1);
        else if (strcmp(sheld, "pedra") == 0 && (strcmp(raj, "lagarto") == 0 || strcmp(raj, "tesoura") == 0))
            printf("Caso #%d: Bazinga!\n", i + 1);
        else if (strcmp(sheld, "lagarto") == 0 && (strcmp(raj, "Spock") == 0 || strcmp(raj, "papel") == 0))
            printf("Caso #%d: Bazinga!\n", i + 1);
        else if (strcmp(sheld, "Spock") == 0 && (strcmp(raj, "tesoura") == 0 || strcmp(raj, "pedra") == 0))
            printf("Caso #%d: Bazinga!\n", i + 1);
        else
            printf("Caso #%d: Raj trapaceou!\n", i + 1);
    }

    return 0;
}

File: 1-iniciante/1828.py
n = int(input())
for i in range(n):
    s, r = str(input()).split()
    g = 'e'
    if r == 'pedra':
        if   s == 'papel':   g = 's'
        elif s == 'tesoura': g = 'r'
        elif s == 'lagarto': g = 'r'
        elif s == 'Spock':   g = 's'
    if r == 'papel':
        if   s == 'pedra':   g = 'r'
        elif s == 'tesoura': g = 's'
        elif s == 'lagarto': g = 's'
        elif s == 'Spock':   g = 'r'
    if r == 'tesoura':
        if   s == 'pedra':   g = 's'
        elif s == 'papel':   g = 'r'
        elif s == 'lagarto': g = 'r'
        elif s == 'Spock':   g = 's'
    if r == 'lagarto':
        if   s == 'pedra':   g = 's'
        elif s == 'papel':   g = 'r'
        elif s == 'tesoura': g = 's'
        elif s == 'Spock':   g = 'r'
    if r == 'Spock':
        if   s == 'pedra':   g = 'r'
        elif s == 'papel':   g = 's'
        elif s == 'tesoura': g = 'r'
        elif s == 'lagarto': g = 's'
        
    print('Caso #{}:'.format(i+1), end=' ')
    if g == 'r': print('Raj trapaceou!')
    elif g == 's': print('Bazinga!')
    else: print('De novo!')
